op-challenger/config/config_test.go

```diff
@@ -99,10 +98,10 @@ func TestCannonAbsolutePreStateRequired(t *testing.T) {
 	require.ErrorIs(t, config.Check(), ErrMissingCannonAbsolutePreState)
 }
 
-func TestCannonDatadirRequired(t *testing.T) {
-	config := validConfig(TraceTypeCannon)
-	config.CannonDatadir = ""
-	require.ErrorIs(t, config.Check(), ErrMissingCannonDatadir)
+func TestDatadirRequired(t *testing.T) {
+	config := validConfig(TraceTypeAlphabet)
```

Comment:
For the other required config values, we use the `TraceTypeCannon`, is there a reason to use the alphabet trace type instead here?

Reply:
It's pretty arbitrary, but the old behaviour was to require it for cannon and not alphabet, so went with alphabet here to prove it was required where it previously wasn't.